pkg/apis/kubic/v1beta1/dexconfiguration_types_test.go

```diff
@@ -27,6 +27,10 @@ import (
 )
 
 func TestStorageDexConfiguration(t *testing.T) {
+	if testing.Short() {
```

Comment:
honestly I just feel like this puts noise into the log yes? every integration test states that it is getting skipped when you run unit tests? I don't know... I don't have a solution, was just a thought. Unless skip does not write to the log. I will have to test this as I have never used it.

Reply:
I agree with @Klaven but I don't know if there is any better solution. Maybe we could perform this check in a file-level initialization function, so `if testing.Short()` then we skip the whole file (and then we could rename this file to someting like `dexconfiguration_types_int_test.go`).

Reply:
I think  golanginit func should do it, if callin